### PR TITLE
First pass of theBoss fixing through new function calls. 

### LINKED ISSUE #512: Boss Assignment fails for members
On server restart the mission commonly fails to assign commander to any member, even when they join the default commander slot, usually this needs manual allocation of commander via debug script.

This sometimes fixes itself when a new member joins, or an existing one DCs and reconnects. 

Either way, this happens regularly enough that a message here is required.

### PR DESCRIPTION
**Pull request by Toakan**
## What type of PR is this.
1. [X] Bug
2. [X] Enhancement

### What have you changed and why?
Information:
    Changed the waitUntil for the Introshot and added / changed the Boss assignment function.

### Please specify which Issue this PR Resolves.
closes #512 

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the Mission in Singleplayer?
2. [X] Have you loaded the Mission in a Dedicated Server?

### Is further testing or are further changes required?
1. [ ] No
2. [X] Yes (Please provide further detail below.)

********************************************************
Notes:
Please test this, i've been unable to completely drop being the boss because it now keeps assigning it to me...

### CODE REVIEW

File: A3-Antistasi/initPlayerLocal.sqf [hunk 1]
```diff
@@ -434,6 +434,7 @@ if (_isJip) then {
 	player setPos (getMarkerPos respawnTeamPlayer);
 }
 else {
+	[3,format ["NOT JIP",_isJip],_filename] call A3A_fnc_log;
```
Comment (wurzel0701) [inline A3-Antistasi/initPlayerLocal.sqf]: `format ["NOT JIP", _isJip]` does not makes sense. It should be either `"Not JIP"` or
```sqf format ["Is JIP: %1, _isJip]```

Comment (Toakan) [reply A3-Antistasi/initPlayerLocal.sqf]: Look at the else statement, it only triggers when the client is not JIP or the _isJip is an unknown.

Comment (wurzel0701) [reply A3-Antistasi/initPlayerLocal.sqf]: In this case you can remove the format too as the string does not any work when only "Not JIP" should be logged

Comment (Spoffy) [reply A3-Antistasi/initPlayerLocal.sqf]: Resolved in 1810638f41aa4dc12546d20867863ccd79152827

File: A3-Antistasi/initPlayerLocal.sqf [hunk 2]
```diff
@@ -374,13 +374,15 @@ if (isMultiplayer) then {
 	};
 };
 
-waitUntil {scriptdone _introshot};
+_introshot spawn { waitUntil { scriptDone _this };};
```
[the remark this answers is not in the document]
Comment (Spoffy) [reply A3-Antistasi/initPlayerLocal.sqf]: Resolved in 1810638f41aa4dc12546d20867863ccd79152827

**Comment by Spoffy** [comment]
Fixed as of 8852b8b2ecd4a3faaef336016fd5f4b2e771c2ca

**Pull request closed without merging**